docs/source/support/node_providers.rst

```diff
@@ -0,0 +1,16 @@
+===========================
+Node as a Service Providers
+===========================
+
+The following staking pools and node infrastructure providers support the NuCypher Network.
+
+This is an opt-in list; providers have not been vetted or endorsed by the core development team. Use your judgement in selecting a provider.
+
+If you are a provider and wish to be included in this list, please submit a pull request.
+
+============= ===
+Provider Name URL
+============= ===
+Provider Name URL
+============= ===
```

Comment:
Let's change this into bullets instead of a table. RST has this stupid requirement where all rows have to have the same width, including the separators (`=====`). This means that if suddenly we have a provider with a long name or URL, they will have to restructure the whole table in their submission.

Reply:
✅